src/app/containers/ConsentBanner/Banner/cookie.amp.jsx

```diff
@@ -0,0 +1,268 @@
+
+const COMMON_HEADING_STYLES = `
+  color: ${C_WHITE};
+  margin-top: ${GEL_SPACING_TRPL};
+  margin-bottom: 0;
+
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+    margin-top: ${GEL_SPACING_QUAD};
+  }
+`;
+
+const Wrapper = styled.div`
+  ${({ service }) => getSansRegular(service)}
+  background-color: ${C_CONSENT_BACKGROUND};
+  max-height: 100vh;
+  padding-right: ${GEL_MARGIN_BELOW_400PX};
+  padding-left: ${GEL_MARGIN_BELOW_400PX};
+
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+    padding-left: ${GEL_MARGIN_ABOVE_400PX};
+    padding-right: ${GEL_MARGIN_ABOVE_400PX};
+  }
+`;
+
+const BannerPage = styled.div`
+  margin-left: auto;
+  margin-right: auto;
+  max-height: 100vh;
+  max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX};
+  overflow-y: auto;
+`;
+
+const Title = styled.h2`
+  ${({ script }) => getDoublePica(script)}
+  ${COMMON_HEADING_STYLES}
+`;
+
+const Heading = styled.h3`
+  ${COMMON_HEADING_STYLES}
+`;
+
+const Paragraph = styled.p`
+  ${({ script }) => getBodyCopy(script)}
+  color: ${C_CONSENT_CONTENT};
+  margin-top: ${GEL_SPACING_DBL};
+  margin-bottom: ${GEL_SPACING_DBL};
+`;
+
+const Link = ({ text, href, className }) => (
+  <a href={href} className={className}>
+    <span>{text}</span>
+  </a>
+);
+
+const StyledLink = styled(Link)`
+  color: ${C_CONSENT_ACTION};
+  text-decoration: none;
+
+  span {
+    border-bottom: ${C_PEBBLE} solid ${BORDER_WIDTH};
+  }
+
+  &:hover,
+  &:focus {
+    background-color: ${C_CONSENT_ACTION};
+    color: ${C_EBON};
+
+    span {
+      border-bottom: transparent solid ${BORDER_WIDTH_TRANSPARENT};
+    }
+  }
+`;
+
+const OptionsList = styled.ul`
+  ${({ script }) => getLongPrimer(script)}
+  align-items: stretch;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  list-style: none;
+  margin-top: 0;
+  margin-bottom: 0;
+  padding-right: 0;
+  padding-bottom: ${GEL_SPACING_DBL};
+  padding-left: 0;
+
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+    flex-direction: row;
+    padding-bottom: ${GEL_SPACING_TRPL};
+  }
+`;
+
+const OptionsItem = styled.li`
+  margin-bottom: ${GEL_SPACING};
+
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+    width: calc(50% - ${GEL_SPACING});
+  }
+
+  button {
+    ${({ service }) => getSansBold(service)}
+    ${({ script }) => getLongPrimer(script)}
+    background: ${C_GHOST};
+    border: none;
+    color: ${C_EBON};
+    cursor: pointer;
+    display: block;
+    height: 100%;
+    min-height: ${MIN_TAP_HEIGHT};
+    padding: ${GEL_SPACING};
+    width: 100%;
+
+    &:hover,
+    &:focus {
+      background-color: ${C_CONSENT_ACTION};
+      color: ${C_EBON};
+      text-decoration: underline;
+    }
+  }
+
+  button[on='tap:AMP.setState({ isManagingSettings: true })'] {
+    background: none;
+    border: ${BORDER_WIDTH} solid ${C_CONSENT_ACTION};
+    color: ${C_CONSENT_ACTION};
+  }
+`;
+
+const AmpCookieBanner = ({
+  id,
+  pages,
```

Comment:
The use of a `pages` prop here implies this component is more generic than it is I think; it may be a little more explicit and pass in the pages in specifically named props.
```suggestion
  initialPage,
  managePage,
```

Reply:
I see your point. I've renamed this prop from `pages` to `translations`.